pkg/backend/openshiftcluster.go

```diff
@@ -76,7 +77,21 @@ func (ocb *openShiftClusterBackend) handle(ctx context.Context, log *logrus.Entr
 	stop := ocb.heartbeat(ctx, cancel, log, doc)
 	defer stop()
 
-	m, err := openshiftcluster.NewManager(log, ocb.env, ocb.db.OpenShiftClusters, ocb.billing, doc)
+	r, err := azure.ParseResourceID(doc.OpenShiftCluster.ID)
+	if err != nil {
+		return err
+	}
```

Comment:
not for the scope of this PR, but I have noticed this pattern to get subscription from ResourceID is used in many places. Would it be better to save it as a standalone field in OpenShiftCluster document? @jim-minter @mjudeikis

Reply:
there's certainly merit in the suggestion.